app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,3 +1,36 @@
+import { renderContactDetails, renderText } from "components/Dashboard/utils";
+
+export const GET_COLUMN_DATA = renderIcon => [
```

Comment:
If it's a function, then it ought to be a util rather than a constant. Here the constant being named with the `GET_` prefix doesn't make sense.